pyfritzhome/fritzhome.py

```diff
@@ -197,6 +197,17 @@ def _get_listinfo_elements(self, entity_type):
         _LOGGER.debug(dom)
         return dom.findall("*")
 
+    def wait_device_txbusy(self, ain, retries=10):
+        """Wait for device to finish command execution."""
+        for _ in range(retries):
+            plain = self.get_device_infos(ain)
+            dom = ElementTree.fromstring(plain)
+            txbusy = dom.findall("txbusy")
+            if txbusy[0].text == "0":
+                return True
+            time.sleep(0.1)
```

Comment:
Do you how long the execution usually takes? In the linked issue people talk about 1-2 seconds. With the values you choose here, they would get false after 1s.
Also I don't know how many calls the fritzbox can handle, especially older models. 
I suggest setting sleep to `0.2`. This should still be fast enough and has a total wait time up to 2s. What do you think?

Reply:
I think the execution will take quite less than one second (_it is an assumption, based on the permanent connection of these devices and that everything longer would be a very poor user experience_). unfortunately I can't test it, since i do not own any of these stationary fritz dect devices. To limit the load on the fritzbox, i've also implemented `get_device_infos` method, which should only query the single device, not all.
Anyway, I agree with increasing to 0.2s, since this should still be ok from a UX perspective :+1: